czech.h

```diff
@@ -1,8 +1,34 @@
+// typy
 #define SOUBOR FILE
-#define když if
 #define plav float
 #define číslo int
+#define znak char
+#define dlouhé long
+#define dlouhý long
+#define krátké short
+#define krátký short
+#define znaménkové sign
+#define znaménkový sign
+#define neznaménkové unsign
```

Comment:
přijde mi zbytečný to mít všechno x-krát
copak sme na hané aby sme měli zneménkové int :D

Reply:
Radši budu mít obecně víc překladů pro jedno slovo než se hádat o jednom, správném překladu

Reply:
tohle ale není překlad, ale skloňování